src/lib/util/generateId/index.ts

```diff
@@ -0,0 +1,11 @@
+function generateId(): string {  
+  if (!window?.crypto?.randomUUID) {
```

Comment:
Thanks for that suggestion, @kimkwanka!

Added a fallback for the edge cases although support is over 90% of browsers, wdyt?

Reply:
Sounds great 👍